.github/workflows/main.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     steps:
       - uses: actions/checkout@v1
       - name: Set up Python
-        uses: actions/setup-python@v1
+        uses: actions/setup-python@v2.0.1
```

Comment:
Is `v2.0.1` preferred over `v2`?

Reply:
I think @dependabot uses the last available released version, while v2 is a "moving" target

Reply:
I prefer the "moving" target. This is just a lot of noise and effort for 0 value.